documentation/asciidoc/computers/compute-module/cmio-display.adoc

```diff
@@ -1,16 +1,16 @@
-== Attach the official 7-inch display
+== Attaching the Touch Display LCD panel
 
 Update your system software and firmware to the latest version before starting.
 Compute Modules mostly use the same process, but sometimes physical differences force changes for a particular model.
 
-=== Connect a display to DISP1
+=== Connect a display to DISP1/DSI1
 
 NOTE: The Raspberry Pi Zero camera cable cannot be used as an alternative to the RPI-DISPLAY adapter. The two cables have distinct wiring.
 
-To connect a display to DISP1:
+To connect a display to DISP1/DSI1:
 
 . Disconnect the Compute Module from power.
-. Connect the display to the DISP1 port on the Compute Module IO board through the 22W to 15W display adapter.
+. Connect the display to the DISP1/DSI1 port on the Compute Module IO board through the 22W to 15W display adapter.
```

Comment:
CM5IO needs J6 added for CAM/DISP1. (That's also missing for connecting a camera further up the page).